src/types.ts

```diff
@@ -854,6 +854,15 @@ export type ReconnectEvent = {
   type: 'reconnect'
 }
 
+/**
+ * The listener connection has been established
+ * note: it's usually a better option to use the 'welcome' event
+ * @public
+ */
+export type OpenEvent = {
```

Comment:
Should this be part of ListenEvent on L878?

Reply:
Good catch! Will add it